src/test/java/com/wikia/webdriver/elements/mercury/components/Head.java

```diff
@@ -0,0 +1,25 @@
+package com.wikia.webdriver.elements.mercury.components;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class Head {
```

Comment:
Please extend WikiBasePageObject and get rid of `driver` parameter and `PageFactory` init  from constructor